WordPress/gutenberg
# Add color block support to latest posts

## What?

Add color block support to the latest posts block.
Partial for https://github.com/WordPress/gutenberg/issues/34647.

## Why?

To allow customizing the colors.

## Testing Instructions

1. Apply the PR
2. Create a couple of posts, some with featured images, that can be used to test the latests posts block.
3. In the block editor, create a new post or page and add a latest posts block.
4. Try the block color options (also in combination with the other block options).
5. Save and view the post or page on the front.
6. confirm that the colors are applied and that the editor and front match.

## Screenshots or screencast 
Editor:
![The block is selected and color options are visible in the block settings sidebar](https://user-images.githubusercontent.com/7422055/175031876-fef322af-0651-4787-b99f-ae1f7dcdce3f.png)

Front
![the latest posts block has a gradient background, text and link colors applied](https://user-images.githubusercontent.com/7422055/175031496-173e072b-1c17-4aff-ade7-d2fb954aee79.png)
